tests/unit/components/Filter/FilterInput.spec.js

```diff
@@ -302,20 +302,20 @@ describe('FilterInput', () => {
         .toHaveBeenNthCalledWith(2, 'text/plain', 'Tag1 Tag2 Foo');
     });
 
-    it('does not copy tags, if they are not selected', () => {
+    it('does not copy tags, if they are not selected', async () => {
       // getSelectionText is mocked to be the input value
-      wrapper.setProps({
+      await wrapper.setProps({
         value: inputValue,
         selectedTags: tags,
       });
-      // copy
-      input.trigger('copy', { clipboardData });
+      const copyEvent = { clipboardData, preventDefault: jest.fn() };
```

Comment:
Did triggering the copy event not work anymore? Just checking

Reply:
It did work, I just tried a workaround here but I fixed it now!